docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,25 @@
+## Using RepoSense with Netlify
+
+[Netlify](https://www.netlify.com) enables you to run RepoSense with your configured files without downloading the latest jar file.
+
+1. Fork the RepoSense repository using this [link](https://github.com/repoSense/RepoSense/fork).
+1. Set up Netlify for your forked repository. We recommend this [site](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) for ease of setup. For the `Build Command` in **Step 5** use:
+`./gradlew run -Dargs="--config ./ --output output_path/  --formats java adoc js"` and for `Publishing Directory` in **Step 5** use `/output_path/reposense-report`
+1. Click on **Settings** in the top, choose **Build & deploy** from the right panel and scroll to **Build hooks**.
+![Build hooks](images/using-netlify-build-hooks.png)
+1. Click **Add build hook**, give your webhook a name, and choose the `master` branch to build. 
+1. Go to your forked repository and click on **Settings**.
```

Comment:
shouldn't this be the `target repo` (the repo to analyze) instead?

What we want is that every time when there is an update in the `target repo`, it will trigger the netlify to start a new build so that they can see the updated result. It shouldn't be the `forked repo` because there won't be any ongoing work over there.

Reply:
I assumed the forked repository to be the target repository and wrote it like that. Will make the change.

Reply:
>If the target repo is different from the forked repo, then the user also needs to update the CSV files in the forked repo, accordingly. Should I include that in the guide too then? I haven't mentioned it till now.

Yes, that needs to be added.